### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by 42CrMo4**
I tried to make meaningful additions and changes for the installation procedure, as i run in this problems myself.

### CODE REVIEW

File: docs/start/install.md [hunk 1]
```diff
@@ -203,7 +213,7 @@ inv server
 For more server options, run:
 
 ```
-inv server -h
+inv server -a
```
Comment (SchrodingersGat) [inline docs/start/install.md:216]: `-h` is the correct option to show help menu. Why the change here?

Comment (42CrMo4) [reply docs/start/install.md:216]: Yes you are right. I misinterpreted the command wrong. I was looking for the option to start the server with a diffrent IP. I'm not very experianced, sorry.

Comment (SchrodingersGat) [reply docs/start/install.md]: No worries :)
